ballerina/commons.bal

```diff
@@ -16,6 +16,47 @@
 
 import ballerina/time;
 
+# Represents the task service.
+public type Service distinct service object {
+  isolated function onTrigger() returns error?;
```

Comment:
For the recurring jobs, we have the task policy to handle errors right? Is that applicable here? If so how does that work

Reply:
Added new API as `onError` to handle them [1f7d01b](https://github.com/ballerina-platform/module-ballerina-task/pull/534/commits/1f7d01b291b90d25b112bc37c5a274e4d3f2ab1e)